app/src/main/java/com/sfvtech/payperview/database/DatabaseHelper.java

```diff
@@ -3,26 +3,30 @@
 import android.content.Context;
 import android.database.sqlite.SQLiteDatabase;
 import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
 
 public class DatabaseHelper extends SQLiteOpenHelper {
 
     public static final String DATABASE_NAME = "viewer_survey.db";
     public static final String LOG_TAG = "DatabaseHelper";
-    private static final int DATABASE_VERSION = 3;
+    private static final int DATABASE_VERSION = 1;
 
```

Comment:
A note here to remember that we'll have to do a fresh install of this app, and make sure the previous versions have uploaded their data beforehand. Otherwise we'll have to migrate the old schema to the new one and that's a pain in the ass.

Reply:
It would be a fresh install anyway because we're building with a different package name?